lib/Wikia/src/Service/Swagger/KubernetesApiProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Wikia\Service\Swagger;
+
+use Wikia\Service\Gateway\KubernetesUrlProvider;
+use Wikia\Util\Statistics\BernoulliTrial;
+
+/**
+ * Temporary class to assist in migration until all services use Kubernetes
+ * @package Wikia\Service\Swagger
+ */
+class KubernetesApiProvider extends ApiProvider {
+	/**
+	 * @Inject({
+	 *   Wikia\Service\Gateway\KubernetesUrlProvider::class,
+	 *   Wikia\Service\Swagger\ApiProviderModule::API_CLIENT_LOG_SAMPLER})
+	 * @param KubernetesUrlProvider $urlProvider
+	 * @param BernoulliTrial $clientLogSampler
+	 */
+	public function __construct(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "__construct" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)

Reply:
Method signature is different, baby.